frameworks/kafka/src/main/java/com/mesosphere/sdk/kafka/api/InterruptProceed.java

```diff
@@ -0,0 +1,63 @@
+package com.mesosphere.sdk.kafka.api;
+
+import com.mesosphere.sdk.scheduler.plan.Phase;
+import com.mesosphere.sdk.scheduler.plan.PlanManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.JSONObject;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/* This is for experimentation. We can use plans/.../ */
+/**
+ * InterruptProceed
+ */
+@Path("/v1")
+@Produces("application/json")
+public class InterruptProceed {
```

Comment:
This whole class is probably not necessary.  You could have the CLI just point to the endpoints already provided by the `PlansResource` class.  If you also want to provide these easy REST API endpoints, this class should either extend or compose `PlansResource`.